CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+public class Jester : IJokeOutput, IJokeService
```

Comment:
```suggestion
public class Jester : IJokeOutput, IJokeService
```
these should be implemented on their own classes, not here.